mm2src/coins/lp_coins.rs

```diff
@@ -1251,17 +1251,21 @@ pub trait WatcherOps {
 
 /// Helper struct wrapping arguments for [TakerCoinSwapOpsV2::send_taker_funding]
 pub struct SendTakerFundingArgs<'a> {
-    /// Taker will be able to refund the payment after this timestamp
-    pub time_lock: u64,
-    /// The hash of the secret generated by taker, this needs to be revealed for immediate refund
+    /// The timestamp after which the taker can refund the funding transaction if the maker hasn't pre-approved the transaction
+    pub funding_time_lock: u64,
+    /// The timestamp after which the taker can refund the payment transaction if the maker hasn't claimed it by revealing their secret
+    pub payment_time_lock: u64,
+    /// The hash of the secret generated by the taker, needed to reveal for immediate refund
     pub taker_secret_hash: &'a [u8],
+    /// The hash of the secret generated by the maker, needed to reveal to spend the payment
+    pub maker_secret_hash: &'a [u8],
     /// Maker's pubkey
     pub maker_pub: &'a [u8],
     /// DEX fee
     pub dex_fee: &'a DexFee,
-    /// Additional reward for maker (premium)
+    /// Additional reward for the maker (premium)
     pub premium_amount: BigDecimal,
-    /// Actual volume of taker's payment
+    /// Actual volume of the taker's payment
```

Comment:
you can either revert this or update similar docs e.g the one below

Reply:
lets revert it ;) https://github.com/KomodoPlatform/komodo-defi-framework/pull/2129/commits/888108ca43530391e6b02dae561407bca01f52c5